pkg/test/server.go

```diff
@@ -49,6 +60,12 @@ func (te *TestEnvironment) Start() (*rest.Config, error) {
 			break
 		}
 		te.ControlPlane = integration.ControlPlane{}
+		if os.Getenv(env_Api) == "" {
+			te.ControlPlane.APIServer = &integration.APIServer{Path: bin_Api}
+		}
+		if os.Getenv(env_Etcd) == "" {
+			te.ControlPlane.Etcd = &integration.Etcd{Path: bin_Etcd}
+		}
```

Comment:
I read the documentation for the testing frameworks https://godoc.org/github.com/kubernetes-sigs/testing_frameworks/integration

Two things:

- We should move the initialization of controlplane before we start the loop.
- To be consistent, we should set the KubeCtl path by setting the environment variable. Either take out the whole kubectl stuff or set the env-variable. Looks like controlplane doesn't give us a way to specify that currently.

Reply:
I wanted to encapsulate one in `TestEnvironment` and initialize it when use (only once in process). This purpose is also hoping no need to set ENV for `TEST_ASSET_KUBECTL`.
It can be done before `ControlPlane.Start()`. 

We can hold this kubeCtl change until really need it at this moment. So, TEST_ENV_KUBECTL is still needed.